src/fondue/Fondue.js

```diff
@@ -712,14 +713,30 @@ export default class Fondue {
 					// inside the same lookup (e.g. 10 alternates for "A", 5 for
 					// "B"), so we keep track of the alternateCount per glyph.
 					subtable.alternateSetOffsets.forEach((_, j) => {
-						currentAllGlyphs[feature.featureTag]["input"].push(
-							letterFor(coverage.glyphArray[j])
-						);
+						if (!coverage.glyphArray) {
+							const range = coverage.rangeRecords[j];
+							if (range) {
+								for (
+									let g = range.startGlyphID;
+									g < range.endGlyphID + 1;
+									g++
+								) {
+									const char = letterFor(g);
+									if (char !== undefined) {
+										parsedLookup["input"].push(char);
+									}
+								}
+							}
```

Comment:
This block seems to very similar to `glyphToLetter`. Is it possible to use that here?

Reply:
While I was figuring out how this would work I thought there were more differences. But looking at the code now I think I can pass the index (`j`) to glyphToLetter and have it work with that.

Reply:
Updated! Much cleaner, thanks for the pointer!